File: baekjoon/silver5/14912.py
# input n, digit
n, digit = map(int, input().split())

count = 0 # frequency of digit

# iterator 1 to n
for i in range(1, n+1):

  # convert int to string
  for a in str(i):
    if int(a) == digit:
      count += 1

# print result
print(count)

File: baekjoon/silver5/14912.cc
#include <iostream>
#include <string>

int main() {
  int n = 0;
  char digit = ' ';
  int count = 0; // frequency of digit

  // input n & digit
  std::cin >> n >> digit;

  // change int to string
  for(int i = 1; i <= n; i++) {
    std::string num = std::to_string(i);

    for(int j = 0; j < num.length(); j++) {
      if (num[j] == digit) {
        count ++;
      }
    }
  }

  // print result ( = count )
  std::cout << count << std::endl;

  return 0;
}

